ddspipe_yaml/src/cpp/YamlReader_types.cpp

```diff
@@ -428,6 +429,80 @@ void YamlReader::fill(
     }
 }
 
+/************************
+* LOGGING CONFIGURATION *
+************************/
+
+template <>
+DDSPIPE_YAML_DllAPI
+void YamlReader::fill<LogFilter>(
+        LogFilter& object,
+        const Yaml& yml,
+        const YamlReaderVersion version /* version */)
+{
+    if (is_tag_present(yml, LOG_FILTER_ERROR_TAG))
+    {
+        object[eprosima::fastdds::dds::Log::Kind::Error] = get<std::string>(yml, LOG_FILTER_ERROR_TAG, version);
+    }
+    if (is_tag_present(yml, LOG_FILTER_WARNING_TAG))
+    {
+        object[eprosima::fastdds::dds::Log::Kind::Warning] = get<std::string>(yml, LOG_FILTER_WARNING_TAG, version);
+    }
+    if (is_tag_present(yml, LOG_FILTER_INFO_TAG))
+    {
+        object[eprosima::fastdds::dds::Log::Kind::Info] = get<std::string>(yml, LOG_FILTER_INFO_TAG, version);
+    }
+}
+
+template <>
+DDSPIPE_YAML_DllAPI
+LogFilter YamlReader::get(
+        const Yaml& yml,
+        const YamlReaderVersion version)
+{
+    LogFilter object;
+    fill<LogFilter>(object, yml, version);
+    return object;
+}
+
+template <>
+DDSPIPE_YAML_DllAPI
+void YamlReader::fill(
+        LogConfiguration& object,
+        const Yaml& yml,
+        const YamlReaderVersion version)
+{
+    // Verbosity optional
+    if (is_tag_present(yml, LOG_VERBOSITY_TAG))
+    {
+        object.verbosity.set_value(get_enumeration<eprosima::fastdds::dds::Log::Kind>(
```

Comment:
You don't need `set_value`. You can simply have a ` == ` and `Fuzzy` will automatically convert it to a `set_value` with a `fuzzy_level_set`.